---
title: "'Finally' no puede aparecer fuera de una instrucción 'Try'"
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 60f02c9000e99c6f1b91fb5f0e2929c4e696e549
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 06/04/2020
ms.locfileid: "84403737"
---
# <a name="finally-cannot-appear-outside-a-try-statement"></a>'Finally' no puede aparecer fuera de una instrucción 'Try'
`Finally` se usa para completar un bloque `Try...Catch...Finally` ; por lo tanto, solo puede aparecer una vez al final del bloque. O bien hay un elemento `Finally`innecesario, o la instrucción `Finally` aparece fuera de los límites de su bloque `Try` correspondiente.  
  
 **Id. de error:** BC30382  
  
## <a name="to-correct-this-error"></a>Para corregir este error  
  
1. Busque y quite la instrucción `Finally` innecesaria.  
  
2. Mueva la instrucción `Finally` a la ubicación adecuada del código.  
  
## <a name="see-also"></a>Consulte también

- [Instrucción Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
